app/models/application_letter.rb

```diff
@@ -46,7 +55,21 @@ def after_deadline?
   # @param none
   # @return [Boolean] true if status changes are allowed
   def status_change_allowed?
```

Comment:
I can understand that methods that for example simply negate a value (like this method before) doesn't have a test (even if in my opinion they should have), but this method isn't that easy to understand for me and I think alone for this reason a test would be good?